components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -755,6 +756,34 @@
 <config_AM_timeSeriesStatsClimatology_reset_intervals>'1000-00-00_00:00:00;1000-00-00_00:00:00;1000-00-00_00:00:00;1000-00-00_00:00:00'</config_AM_timeSeriesStatsClimatology_reset_intervals>
 <config_AM_timeSeriesStatsClimatology_backward_output_offset>'00-03-00_00:00:00'</config_AM_timeSeriesStatsClimatology_backward_output_offset>
 
+<!-- AM_timeSeriesStatsMonthlyMax -->
+<config_AM_timeSeriesStatsMonthlyMax_enable>.false.</config_AM_timeSeriesStatsMonthlyMax_enable>
+<config_AM_timeSeriesStatsMonthlyMax_compute_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMax_compute_on_startup>
+<config_AM_timeSeriesStatsMonthlyMax_write_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMax_write_on_startup>
+<config_AM_timeSeriesStatsMonthlyMax_compute_interval>'00-00-00_01:00:00'</config_AM_timeSeriesStatsMonthlyMax_compute_interval>
+<config_AM_timeSeriesStatsMonthlyMax_output_stream>'timeSeriesStatsMonthlyMaxOutput'</config_AM_timeSeriesStatsMonthlyMax_output_stream>
+<config_AM_timeSeriesStatsMonthlyMax_restart_stream>'timeSeriesStatsMonthlyMaxRestart'</config_AM_timeSeriesStatsMonthlyMax_restart_stream>
```

Comment:
this default needs to be 'none' as well

Reply:
We have scripts that auto-generate the namelist defaults and use the settings in Registry when there is no conflict with the E3SM settings. Because these are all new variables, the scripts picked up the Registry defaults and they ended up here. I've changed the restart_stream for both MonthlyMin and MonthlyMax to 'none' as well as enabled them by default -- to match the Monthly stream.